src/app/components/navbar/navbar.component.html

```diff
@@ -24,13 +24,32 @@
     </ng-container>
 
     <div class="toolbar-spacer"></div>
+
     <div [hidden]="(currentExperiment|async) == null">
         <span>{{currentExperiment|async}}</span>
         <a class="navigation-link" mat-icon-button [hidden]="(currentExperiment|async) == null" [routerLink]="['/']"
             aria-label="change experiment">
             <mat-icon>swap_horiz</mat-icon>
         </a>
     </div>
+    <button class="downloads-button" mat-button [matMenuTriggerFor]="downloadList">
```

Comment:
Maybe this icon should only appear if there is any download. If the icon is always visible, then the menu should have a short text explaining that downloads will show up here if there are any.
This button could display some animation if there is any preparing/ongoing download (and stop the animation if all downloads are ready for download).